cli/src/semgrep/run_scan.py

```diff
@@ -635,13 +635,15 @@ def run_scan(
                     baseline_target_strings = target_strings
                     baseline_target_mode_config = target_mode_config
                     if target_mode_config.is_pro_diff_scan:
```

Comment:
Minor note that we should change references like these to "interfile_diff_scan"

Reply:
added a todo note.